docs/structured-streaming-kafka-integration.md

```diff
@@ -416,6 +416,24 @@ The following configurations are optional:
 </tr>
 </table>
 
+### Consumer Caching
+
+It's time consuming to initialize Kafka consumers especially in streaming scenarios where processing time is key factor.
```

Comment:
time-consuming
consumers, especially
is _a_ key factor

Reply:
Fixed.